src/modules/Auth/components/ButtonMenu.tsx

```diff
@@ -0,0 +1,95 @@
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import Logout from '@mui/icons-material/Logout';
+import Settings from '@mui/icons-material/Settings';
+import Box from '@mui/material/Box';
+import IconButton from '@mui/material/IconButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Tooltip from '@mui/material/Tooltip';
+import { makeStyles } from '@mui/styles';
+import * as React from 'react';
+
+const useStyles = makeStyles(() => ({
+  logOut: {
+    color: '#FF0000',
+  },
+}));
+
+export const ButtonMenu = () => {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  return (
+    <>
+      <Box sx={{ display: 'flex', alignItems: 'center', textAlign: 'center' }}>
+        <Tooltip title="Account settings">
+          <IconButton
+            onClick={handleClick}
+            size="small"
+            sx={{ ml: 2 }}
+            aria-controls={open ? 'account-menu' : undefined}
+            aria-haspopup="true"
+            aria-expanded={open ? 'true' : undefined}
+          >
+            <ArrowDropDownIcon />
+          </IconButton>
+        </Tooltip>
+      </Box>
+      <Menu
+        anchorEl={anchorEl}
+        id="account-menu"
+        open={open}
+        onClose={handleClose}
+        onClick={handleClose}
+        PaperProps={{
+          elevation: 0,
+          sx: {
+            overflow: 'visible',
+            filter: 'drop-shadow(0px 2px 8px rgba(0,0,0,0.32))',
+            mt: 4.5,
```

Comment:
As you can see the spacing coefficient in the theme is 4 px. It means that if you set the value of css properties which can be measured in pixels this value multiple to this coefficient. So if you set 2 the value is 8px, 3 - 12 px and etc.
Of course you can set the float value but I advise to use only integer values if you want mui transform it to pixel by multiplying to the coefficient. If you need the value which not multiple of 4 I would set the explicit value with units In this case it '18px'.
![image](https://user-images.githubusercontent.com/40975346/153462376-24eb79be-6a7c-425a-afda-1d61dd83fafa.png)

Reply:
And can you check all places where you pass the float numbers and update its to explicitly values with units